internal/runtime/cpu_quota_linux.go

```diff
@@ -41,9 +41,14 @@ func CPUQuotaToGOMAXPROCS(minValue int) (int, CPUQuotaStatus, error) {
 		return -1, CPUQuotaUndefined, err
 	}
 
+	if cfg.Reserved > 0 {
+		quota -= cfg.Reserved
+	}
+
 	maxProcs := int(math.Ceil(quota))
```

Comment:
Do we want `Floor` here after all? If we use `Ceil`, we will not be respecting the full reservation for the `Reserved` quota. Can `Reserved` ever be fractional?

Reply:
Correct, we do want floor, this PR was just written orthogonally to #13.